### PR TITLE
Fixed small contact.html error. Got saved url's from servlet and used…

### PR DESCRIPTION
**Pull request by Jaxobc**
… javascript to display them in my comment section. Changed styling of objects with CSS.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -53,6 +53,17 @@ function createCommentElement(comment) {
 
   commentElement.appendChild(nameElement);
   commentElement.appendChild(textElement);
+
+  imageURL = comment.imageURL;
+  if(imageURL != null){
+    console.log("Vibes");
+    const imgElement = document.createElement('IMG');
```
Comment (kennzoid) [inline portfolio/src/main/webapp/script.js:60]: nit: Tags should be lowercase.

https://google.github.io/styleguide/htmlcssguide.html#Capitalization

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -53,6 +53,17 @@ function createCommentElement(comment) {
 
   commentElement.appendChild(nameElement);
   commentElement.appendChild(textElement);
+
+  imageURL = comment.imageURL;
+  if(imageURL != null){
+    console.log("Vibes");
```
Comment (kennzoid) [inline portfolio/src/main/webapp/script.js:59]: Consider using more descriptive logging if you are going to keep this in production code.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (Jaxobc) [reply portfolio/src/main/webapp/script.js:60]: Okay sure, I've amended it.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (Jaxobc) [reply portfolio/src/main/webapp/script.js:59]: Thanks Kenneth. I don't need it in my production code, so I'll remove it.

**Pull request merged**